src/relayserver/RelayServer.ts

```diff
@@ -658,32 +658,32 @@ latestBlock timestamp   | ${latestBlock.timestamp}
   }
 
   /**
-   * Resend the earliest pending transactions of all signers (manager, workers)
-   * @return the receipt from the first request
+   * Resend all outgoing pending transactions with insufficient gas price by all signers (manager, workers)
+   * @return the mapping of the previous transaction hash to details of a new boosted transaction
    */
-  async _boostStuckPendingTransactions (blockNumber: number): Promise<PrefixedHexString[]> {
-    const transactionHashes: PrefixedHexString[] = []
+  async _boostStuckPendingTransactions (blockNumber: number): Promise<Map<PrefixedHexString, SignedTransactionDetails>> {
```

Comment:
I agree that txHash is not a viable ID field, especially in the context of boosting. 
I am not sure changing that is in a scope of this PR though. You can notice that previously the returned transactions were not identified at all, and the only current use for this mapping is logging and testing.
I start feeling like our storage could benefit from some SQL or whatnot, don't you think?

Reply:
The problem with using address+nonce (which is a key in the current ✌"DB"✌ by  the way) is that it forces it to forget older versions of the transactions. It can hurt us, as I have personally experienced boosting a transaction on mainnet only to find out the old one actually got mined but apparently was not picked up by infura for some time, or something like that.

Reply:
https://opengsn.atlassian.net/browse/OG-317
Unique static TX identifier in Server database with list of transactions boosts 'table'

Reply:
ok. commented there.